docs/source/reference/yaml-spec.rst

```diff
@@ -222,6 +223,71 @@ You can also specify a specific region, zone, or Kubernetes context.
     infra: k8s/my-h100-cluster-context
 
 
+.. _yaml-spec-resources-autostop:
+
+``resources.autostop``
+~~~~~~~~~~~~~~~~~~~~~~
+
+Autostop configuration (optional).
+
+Controls whether and when to automatically stop or tear down the cluster after it becomes idle. See :ref:`auto-stop` for more details.
+
+Format:
+
+- ``true``: Use default idle minutes (5)
+- ``false``: Disable autostop
+- ``<num>``: Stop after this many idle minutes
+- ``<num><unit>``: Stop after this much time
+- Object with configuration:
+  - ``idle_minutes``: Number of idle minutes before stopping
+  - ``down``: If true, tear down the cluster instead of stopping it
+
+``<unit>`` can be one of:
+
+- ``s``, ``sec``: seconds
+
+- ``m``, ``min``: minutes
+
+- ``h``, ``hr``: hours
+
+- ``d``, ``day``: days
+
+.. maybe instead of warning, have it be a note?
```

Comment:
We can delete the inline comment and just make a comment on the PR so reviewers can see. IMO note or warning is fine either way.
I kind of feel like we shouldn't allow the use of seconds, if they will always be rounded away.

Reply:
have seconds allowed in case anything else that we add units will use seconds so the same function can be used for all of them